Specification/1._Introduction.md

```diff
@@ -4,23 +4,19 @@ permalink: /Specification/1._Introduction/
 
 # Introduction
 
-This section covers basic concepts that govern development and maintenance of the specification.
+This section covers the basic concepts that govern development and maintenance of the specification.
 The actual specification is contained in the following sections.
 
 ## Core design goals
 
-The list of core design goals below helps to understand the basic concepts of UAVCAN and the motivation behind them.
+The list of core design goals below help explain the basic UAVCAN concepts and the motivation behind them.
 
-* **Democratic network** - since the network does not require a master node (bus controller),
-it has no single point of failure.
-* **Nodes shall be able to exchange long payloads** - typical use cases for a vehicle bus include the need to transfer
-sets of related parameters (e.g., GNSS solution, 3D vectors, etc.), where each parameter cannot be used independently.
-Such a set of parameters often does not fit into a single CAN frame,
-hence the need to split it into several CAN frames with a subsequent reassembly process on the receiving nodes.
-* **Support for redundant interfaces and redundant nodes** - it is a common requirement for
+* **Democratic network** - All nodes in the network should have the same communication rights so that there is no single point of failure.
+* **Nodes can exchange long payloads** - Nodes should be able to transfer large sets of related parameters (e.g., GNSS solution, 3D vectors, etc.) in a single frame. This avoids the computational and programming cost required to split messages into separate frames and then reassemble them at receiving nodes.
```

Comment:
Thanks for the clarification. Love your text, but removing the "efficient" because the key here is simple (efficient is implied and covered by a later point).